src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -1,321 +1,30 @@
 package uk.gov.justice.digital.hmpps.whereabouts.services.court
```

Comment:
Old CourtService renamed VideoLinkBookingService.  
New CourtService here has nothing in common with old CourtService.